requirements/lint.txt

```diff
@@ -1,8 +1,13 @@
 -r pip.txt
 maxcdn
 astroid
-pylint
-# prospector==0.12.6 currently has issues with pydocstyle
```

Comment:
I unppined this because I didn't find any error related with `pydocstyle`. Besides, I think the new version works as it should.